keyboard/ergodox_ez/Makefile

```diff
@@ -88,19 +88,19 @@ OPT_DEFS += -DBOOTLOADER_SIZE=512
 
 
 # Build Options
-#   comment out to disable the options.
+#   Set to yes to enable by default, or make MIDI_ENABLE=yes or
+#   make SLEEP_LED_ENABLE=no to override for a given make invocation
 #
-BOOTMAGIC_ENABLE = yes # Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE  = yes # Mouse keys(+4700)
-EXTRAKEY_ENABLE  = yes # Audio control and System control(+450)
-# CONSOLE_ENABLE   = yes # Console for debug(+400)
-COMMAND_ENABLE   = yes # Commands for debug and configuration
-CUSTOM_MATRIX    = yes # Custom matrix file for the ErgoDox EZ
-SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
-NKRO_ENABLE      = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-# MIDI_ENABLE    = yes 		# MIDI controls
-# UNICODE_ENABLE = yes 		# Unicode
-
+BOOTMAGIC_ENABLE := yes # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE  := yes # Mouse keys(+4700)
```

Comment:
@ezuk I just stumbled across this PR. However, my Google-fu turns up [this useful StackOverflow question](http://stackoverflow.com/questions/448910/makefile-variable-assignment). It seems to be a question of lazy/immediate evaluation, but I'm not sure how it specifically applies here.